x-pack/plugin/sql/qa/src/main/java/org/elasticsearch/xpack/sql/qa/SqlProtocolTestCase.java

```diff
@@ -244,22 +263,21 @@ private void assertQuery(String sql, String columnName, String columnType, Objec
             options.addHeader("Accept", randomFrom("*/*", "application/" + format));
             request.setOptions(options);
         }
-        if ((false == columnar && randomBoolean()) || columnar) {
+        if (columnar || randomBoolean()) {
```

Comment:
@astefan, Do you remember the purpose of the previous status of this if?

Reply:
@matriv, yes. The condition says "set the columnar parameter to true, if columnar value is true; set the columnar parameter to false **explicitly** if randomBoolean is true or do nothing (columnar remains at the default value - false) if randomBoolean is false". And I believe @bpintea 's simplification is correct.